Makefile

```diff
@@ -76,22 +76,22 @@ go_webdriver_test: go_webdriver_deps
 			--frame_buffer=$(USE_FRAME_BUFFER)
 	$(STOP_XVFB)
 
-web_components_test: webdriver_deps web_component_tester
+web_components_test: xvfb firefox chrome web_component_tester webserver_deps
```

Comment:
Hmm, can we directly depend on `node-web-component-tester`? Just curious, not a big deal.